src/components/Header.tsx

```diff
@@ -0,0 +1,7 @@
+import React from 'react'
+
+const Header: React.FC = () => {
+	return <h1>Test</h1>
+}
+
+export default Header
```

Comment:
Do we want to use `default` exports for components? :thinking:

Reply:
The only resource commenting on this issue is called TypeScript **DEEPDIVE**. And it's quite honestly diving way too deep for me to understand about every second word. What I get from it is that `export default` can cause problems in refactoring and other areas and therefore `import { Foo } from "./foo";` somehow makes more sense to some people for some reasons than `import Foo from "./foo";` so _curly brakes good, without brackets bad_.

...but I don't understand why, when I  finally manage to make my one arrow function after several hours of research and thinking and experimenting, junior needs to take into consideration CommonJS interop, Dynamic imports or what "Needs two lines for non-class / non-function" even means. I don't want to take it lightly or "run" from new tasks but there's so many tasks this just overwhelms me.

[https://basarat.gitbook.io/typescript/main-1/defaultisbad](url)

Is this really important for us to learn and understand when in the end it's a convention people either use or don't and some seniors find it slightly problematic because CommonJS interop reasons? I find it outside of the scope of things we're learning and I see it as too high effort task to prioritise it when our other tasks are "try to understand how to write CSS correctly". :sweat_smile:

Reply:
OK, let's say, that using `default` exports is general practise and leave it at that. :smile: :+1: